src/main/java/emissary/place/CoordinationPlace.java

```diff
@@ -108,6 +108,8 @@ protected void configurePlace() {
         updateTransformHistory = configG.findBooleanEntry("UPDATE_TRANSFORM_HISTORY", false);
 
         placeKeys = configG.findEntries("SERVICE_COORDINATION");
+        placeKeys.addAll(configG.findEntries("APPEND_SERVICE_COORDINATION"));
+        placeKeys.removeAll(configG.findEntries("DENY_SERVICE_COORDINATION"));
```

Comment:
As implemented, "DENY_SERVICE_COORDINATION" is final;  we can't override it in another flavor file.  Is that what we want?

Are we not implementing a "REMOVE_SERVICE_COORDINATION" option since we could just use `SERVICE_COORDINATION != "PLACE_TO_DENY"?